tests/test_market_redis_connection.py

```diff
@@ -208,7 +208,6 @@ def test_subscribes_to_market_channels(self):
                 "id/DELETE_BID": self.subscriber._delete_bid,
                 "id/ACCEPT_BID": self.subscriber._accept_bid,
                 "id/BID": self.subscriber._bid,
-                "id/CLEAR": self.subscriber._clear_market,
```

Comment:
I see you're removing this functionality. What was it doing and why is it not required anymore? Maybe I missed the discussion in the slack channel.

Reply:
Yes, the only logic inside this _clear_market function is a call to a deprecated `match_offers_bids` which doesn't carries implementation
I removed the `match_offers_bids` function and this wrapper one too 
[you can check the conversation](https://gridsingularity.slack.com/archives/C01G89NENQZ/p1624285347002000)

Reply:
Great, thanks!